index.html

```diff
@@ -4,6 +4,8 @@
     <link rel="stylesheet" type="text/css" href="static/styleguide.css">
     <script src="static/styleguide.js"></script>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700,300' rel='stylesheet' type='text/css'>
```

Comment:
Do we need this line? I see it's already added in the sass files and it's currently using that font in the github page.

https://github.com/ombulabs/styleguide/blob/44ee54f2943c9366f679cb6e5597e4ada614e705/vendor/assets/stylesheets/ombulabs/styleguide.scss#L7

Maybe we need this for the icons to work though. Not a change request, just to know.